backend/src/seeds/seed-types/seed-data.type.ts

```diff
@@ -26,6 +30,8 @@ type Resume = {
     created_at: Date;
     updated_at: Date;
     deleted_at: Date;
+    user_id: string;
+    template_id: string;
```

Comment:
why does profile need template_id

Reply:
it is for resume, not profile. 
please expand hidden rows